docker/kubernetes-tentacle/bootstrapRunner/bootstrapRunner.go

```diff
@@ -88,7 +87,7 @@ func Write(stream string, text string, counter *SafeCounter) {
 	//https://go.dev/tour/concurrency/9
 	counter.Mutex.Lock()
 
-	fmt.Printf("%d|%s|%s|%s\n", counter.Value, time.Now().UTC().Format(time.RFC3339Nano), stream, text)
+	fmt.Printf("|%d|%s|%s\n", counter.Value, stream, text)
```

Comment:
Remove our own timestamp - rely on the API Server's timestamp instead (it gets added at the front with a space character

Reply:
is it in the correct format/resolution?

Reply:
Good question - looks like it - I have a temp build that prints "Parsing line: ..."

![CleanShot 2024-04-10 at 17 22 44@2x](https://github.com/OctopusDeploy/OctopusTentacle/assets/2888279/5cb603b4-91c8-44dc-8f14-16192012d1c9)